spec/factories/legal_aid_applications.rb

```diff
@@ -69,6 +71,43 @@
       end
     end
 
+    # The trait :with_scope_limitations will set up:
+    #
+    # * one proceeding type
+    # * a default substantive scope limitation for that proceeding type
+    # * a default delegated functions scope limitation for that proceeding type
+    # * add the default substantive scope limitation to the application
+    # * add the default delegated functions scope limitation to the application (SEE NOTE BELOW)
+    #
+    # NOTE: The delegated function scope limitation will only be added to the application if
+    # the :with_delegated_functions trait is specified BEFORE the :with_scope_limitations trait
+    # when created, e.g.
+    #
+    #     create :legal_aid_application, :with_delegated_functions, :with_scope_limitations
+    # will work.
+    #
+    #    create :legal_aid_application, :with_scope_limitations, :with_delegated_functions
+    # will NOT work
+    #
+    # See factory specs at spec/factory_specs/legal_aid_application_factory_spec.rb
+    #
```

Comment:
Nice comment 👍

Reply:
No need for the comment now - have refactored along the lines suggested by Rob which means that theere isn't the need to make sure the traits are specified in the right order.